### ISSUE TITLE
Support tickets should automatically close when without user interaction

### ISSUE THREAD

**Issue by MidSpike** [Inertia-Lighting/discord-bot-api-server#64]
![image](https://user-images.githubusercontent.com/22574513/121227074-2f87b700-c859-11eb-86b8-98de1066b01d.png)
The image above is a suggested metric for determining whether or not a support ticket has been thoroughly interacted with.
I believe that if a user hasn't said `done` or `cancel` (as shown above) within 30 minutes of a support ticket opening, the ticket should automatically close.

**Comment by MidSpike** [comment]
@rossloika @itsdrawn I want your input on this issue

**Comment by itsdrawn** [comment]
@rossloika please state your input on this issue

**Comment by rossloika** [comment]
Good idea, it would allow more ease to our support team and allow more focused interactions to tickets that are active and are in need of assistance. 👍

**Comment by MidSpike** [comment]
Fixed in #69
